beacon/goclient/types.go

```diff
@@ -9,5 +13,5 @@ var (
 	EpochsPerSyncCommitteePeriod         uint64       = 256
 	TargetAggregatorsPerCommittee        uint64       = 16
 	FarFutureEpoch                       phase0.Epoch = 1<<64 - 1
-	GenesisForkVersion                                = phase0.Version{0x0, 0x0, 0x10, 0x20} // prater
+	GenesisForkVersion                                = networkconfig.PraterNetwork.ForkVersion // TODO: pass from caller
```

Comment:
So this could ideally look like this.
```go
GenesisForkVersion = networkconfig.ForkVersion
```
and when initializing we'll do something like 
```go
networkconfig.SetNetwork(PraterNetwork)
```
where `PraterNetwork` is a struct holding all values incl `ForkVersion`